fio_wrapper/Dockerfile

```diff
@@ -1,12 +1,12 @@
 FROM registry.access.redhat.com/ubi8:latest as builder
 COPY image_resources/centos8.repo /etc/yum.repos.d/centos8.repo
 RUN dnf install -y --enablerepo=centos8 make gcc libaio zlib-devel libaio-devel
-RUN curl -L https://github.com/axboe/fio/archive/fio-3.12.tar.gz | tar xzf -
-RUN pushd fio-fio-3.12 && make -j2
+RUN curl -L https://github.com/axboe/fio/archive/fio-3.19.tar.gz | tar xzf -
+RUN pushd fio-fio-3.19 && make -j2
 
 FROM registry.access.redhat.com/ubi8:latest
-COPY --from=builder /fio-fio-3.12/fio /usr/local/bin/fio
-RUN dnf install -y --nodocs git python3-pip python3-requests python3-numpy libaio zlib procps-ng iproute net-tools ethtool nmap iputils && dnf clean all
+COPY --from=builder /fio-fio-3.19/fio /usr/local/bin/fio
+RUN dnf install -y --nodocs python3-pip python3-requests python3-numpy libaio zlib procps-ng iproute net-tools ethtool nmap iputils && dnf clean all
```

Comment:
as @dry923 pointed out about the missing git ? ci passed fine so maybe its installed by default on ubi8?

Reply:
I don't know why git was there. It's not required to run the workload.

Reply:
it was there because of the old way of cloning snafu, now we dont clone snafu anymore we just copy it